pkgs/development/python-modules/fast-histogram/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, pytestCheckHook
+, fetchFromGitHub
+, pythonOlder
```

Comment:
```suggestion
, pythonOlder
, python
```

Sorry, `nixpkgs-hammer` erroneously flagged this as unused

Reply:
Ok, I added.